bridge_adaptivity/bridge_lti/provider.py

```diff
@@ -144,17 +144,17 @@ def create_sequence_item(request, sequence, start_activity, tool_provider, lti_c
     return sequence_item
 
 
-def announcement_page(request):
+def stub_page(request, title=None, message=None, tip=None):
```

Comment:
Please seat meaningful value for default arguments instead None

Reply:
Nice catch, thanks!